src/TextBox.js

```diff
@@ -520,14 +518,29 @@ function(d) {
 
   /**
       @memberof TextBox
-      @desc Toggles the ability to render simple HTML tags. Currently supports `<b>`, `<strong>`, `<i>`, and `<em>`.
+      @desc Toggles the ability to render simple HTML tags. Defaults to supporting `<b>`, `<strong>`, `<i>`, and `<em>`.
       @param {Boolean} [*value* = true]
       @chainable
   */
   html(_) {
     return arguments.length ? (this._html = _, this) : this._html;
   }
 
+  /**
+      @memberof TextBox
+      @desc Provides the replacment style for html tags. Defaults to  `<b>`, `<strong>`, `<i>`, and `<em>`.
+      @param {Dictionary} [*value* =  {
+                i: 'font-style: italic;',
+                em: 'font-style: italic;',
+                b: 'font-weight: bold;',
+                strong: 'font-weight: bold;'
+            }]
+      @chainable
+*/
+  tagStyling(_) {
```

Comment:
Let's rename `this._tagLookup` to `this._htmlLookup`, and then rename this method `tagStyling` to `htmlStyling` to match.

Reply:
changed: thanks